### ISSUE TITLE
Doesn't work with bluetooth headset controls

### ISSUE THREAD

**Issue by maxenceleduc92** [carlin-q-scott/browser-media-keys#151]
Hi,
First, thanks for this module and all the work you've been doing.

I noticed the media keys on my keyboard (CM Storm Trigger Z) work perfectly and it's awesome, but it won't work with my Plantronics Backbeat Pro 2 media controls. It works fine with Spotify though, so it means the computer receives the play/pause command correctly, but it won't work on youtube videos with this module.

Thank you!

**Comment by SonicGold** [comment]
Are you on Mac OS ? You might need this to ensure your play/pause buttons work correctly across various music apps : https://github.com/jguice/mac-bt-headset-fix

**Comment by maxenceleduc92** [comment]
No, I'm on Windows. As I said, it works fine with Spotify so the problem comes from this extension
